views/extras.tmpl

```diff
@@ -196,23 +196,6 @@
   {{- end -}}
 {{end}}
 
-{{define "listViewRouting"}}
-	<div class="fs12 nowrap text-left" style="margin:auto auto auto 0px;">
-		<select
-		  class="form-control-sm mb-2 mr-sm-2 mb-sm-0"
-		  data-target="pagenavigation.listview"
-		  data-action="change->pagenavigation#setListView"
-		  >
-			<option {{if eq . "years"}}selected {{end}}value="years">Years</option>
-			<option {{if eq . "months"}}selected {{end}}value="months">Months</option>
-			<option {{if eq . "weeks"}}selected {{end}}value="weeks">Weeks</option>
-			<option {{if eq . "days"}}selected {{end}}value="days">Days</option>
-			<option {{if eq . "windows"}}selected {{end}}value="ticketpricewindows">Windows</option>
-			<option {{if eq . "blocks"}}selected {{end}}value="blocks">Blocks</option>
-		</select>
-	</div>
-{{end}}
-
```

Comment:
this may not be related to the task, just found an unused template and removed it